pjlib/include/pj/sock.h

```diff
@@ -277,7 +282,7 @@ extern const pj_uint16_t PJ_IPTOS_MINCOST;
 
 
 /**
```

Comment:
Looks like pjsip isn't consistent with its commenting style between files. It's probably still better to match the style of the file though.